### PR TITLE
Add docker-compose to run as standalone container

### LINKED ISSUE #31: Sunsynk Docker
I LOVE this addon. The only downside is it require Home Assistant Supervised installation. On my Synology it mean you have install a virtual machine which is resource consuming.

It's possible to run this via Docker Container?

Thank you.

### PR DESCRIPTION
**Pull request by jpmeijers**
Fixes #31 

* Added docker-compose to run both sunsynk-multi as well as mbusd
* Cleaned up sunsynk-multi Dockerfile, as list of Python dependencies should be in a requirements.txt file

### CODE REVIEW

File: hass-addon-sunsynk-multi/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+paho-mqtt
+mqtt-entity
+sunsynk[pymodbus,umodbus]
+pyyaml
```
[the remark this answers is not in the document]
Comment (kellerza) [reply hass-addon-sunsynk-multi/requirements.txt:4]: Always keep the versions pinned, lots of issues can be introduced when a new library is released. This way it's deterministic across all installs.

Also `script\copy2local.cmd` uses the comments & info in the Dockerfile when I test local addon instances, so let's keep the Dockerfile exactly as is.

**Review by kellerza — COMMENTED** [review]
Thanks! What do you use this for if I may ask, do you deploy it locally without HASS OS?

If you run it locally, maybe the Text in README.me should rather be in/at the end of [getting-started](https://github.com/kellerza/sunsynk/blob/main/www/docs/guide/getting-started.md) - this is the source for the webpage/help

Another way to do the settings it to load a yaml file - today it's called `.local.yaml`, but you can also call it `config.yaml` by changing [this](https://github.com/kellerza/sunsynk/blob/main/hass-addon-sunsynk-multi/options.py#LL85C49-L85C62) - yaml is a bit easier to edit manually (and most HASS people are used to yaml spacing)

File: hass-addon-sunsynk-multi/requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (jpmeijers) [reply hass-addon-sunsynk-multi/requirements.txt:4]: Can we really not move the libraries to a `requirements.txt`? That is how it is normally done.

Looking at your copy2local, it looks hardcoded for your use. Maybe we can even simplify it if we have the dependencies in a separate requirements.txt file.

This is however a different issue than the one this PR is trying to solve. I'll remove those changes from this branch and move it to another branch.

**Comment by jpmeijers** [comment]
> Thanks! What do you use this for if I may ask, do you deploy it locally without HASS OS?

Indeed. I find it really annoying how HASS OS / HA Supervisor hijacks your entire machine. I've been through many different setups where the host machine is shared between Home Assistant and other services, and each time either HA is unhappy, or it kills my other containers. Yesterday after a reboot HA didn't want to start up, so I gave up and decided to go clean HA Core in a container, and run all the add-ons manually in containers next to it.

I prefer running everything in docker containers on my server, allowing me to update the host without affecting any of the running services. So very similar reason as pinning library versions. Having to run a VM for HA is a lot of unnecessary overhead.

Regarding this PR: I'll test the yaml config file approach, apply your requested changes and push a new commit.

**Comment by kellerza** [comment]
Thank you @jpmeijers 🚀

**Comment by Mountaingoat66** [comment]
Can I ask a question here?

**Comment by kellerza** [comment]
> Can I ask a question here?

You can, but always better to ask in a new/active discussion, or new/open issue (like #183)

**Pull request merged**